plugin/trino-hudi/pom.xml

```diff
@@ -15,7 +15,8 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.hudi.version>0.12.3</dep.hudi.version>
+        <dep.hudi.version>0.14.0</dep.hudi.version>
+        <dep.kryo.shaded.version>4.0.2</dep.kryo.shaded.version>
```

Comment:
No need to add a version property when there is only one dependency that uses it. We add properties when there are multiple dependencies that all need to share the same version.

Reply:
Got it.  Fixed now.